taipy/core/notification/notifier.py

```diff
@@ -118,6 +128,18 @@ def register(
     def unregister(cls, registration_id: str):
         """Unregister a listener.
 
+        Example usage:
+
+        ```python
+        registration_id, registered_queue = Notifier.unregister(
+            entity_type=EventEntityType.CYCLE,
+            entity_id="CYCLE_cycle_1",
+            operation=EventOperation.CREATION
+        )
+
+        Notifier.unregister(registration_id)
+        ```
```

Comment:
This is not correct. we only provide the registration_id to unregister.

Reply:
I believe that's the example no?

Reply:
On line 141, it should be `register`. That's why.

Reply:
ahh fixed! thanks!